<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>apprtcwinClass</class>
 <widget class="QMainWindow" name="apprtcwinClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>986</width>
    <height>664</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>apprtcwin</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QFrame" name="frame_2">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QFrame" name="frame_3">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>本地</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="MyGlView" name="localView" native="true">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame_4">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>远端</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="MyGlView" name="remoteView" native="true">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frame">
      <property name="maximumSize">
       <size>
        <width>150</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLineEdit" name="roomEdit">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>20</y>
         <width>131</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string>请输入房间号</string>
       </property>
      </widget>
      <widget class="QPushButton" name="testBtn">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>60</y>
         <width>61</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>进入
房间</string>
       </property>
      </widget>
      <widget class="QPushButton" name="testBtn2">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>60</y>
         <width>61</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>离开
房间</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="msgEdit">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>190</y>
         <width>131</width>
         <height>51</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="testBtn3">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>260</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>发送post</string>
       </property>
      </widget>
      <widget class="QPushButton" name="testBtn4">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>320</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>发送wss</string>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>986</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>MyGlView</class>
   <extends>QWidget</extends>
   <header>MyGlView.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="apprtcwin.qrc"/>
 </resources>
 <connections/>
</ui>
